xapian-applications/omega/templates/query

```diff
@@ -52,50 +52,55 @@ document.write("<span>Modified:<time datetime=\""+D+" "+T+"\"><b>"+D+"</b></time
 <body>
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
+<div class="query-container">
 ${Search Control(Start) - shows search button, options and textbox}
-<input id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
+<input class="query-textbox" id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
```

Comment:
Make that argument when you have CSS that justifies it, and keep things lean until then. Add classnames or ids when doing the semantic HTML pass only if they improve the semantics of the HTML.